verification/curator-service/ui/cypress/fixtures/bad_bulk_data.csv

```diff
@@ -0,0 +1,2 @@
+sourceId,sourceEntryId,sex,ageRangeStart,ageRangeEnd,country,locationQuery,latitude,longitude,geoResolution,locationName,dateConfirmedStart,dateConfirmedEnd,url
+abc,def,InvalidSex,42,42,Canada,Canada,42,84,Country,Canada,06/23/2020,06/23/2020,foo.bar
```

Comment:
Should we have more than one line of data to test the bulk-ness of it? Same below.

Reply:
Good idea, will add.